website_hr_job_unpublication/__manifest__.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    'name': 'Website Hr Job Unpublication',
+    'summary': """
+        Allow employees to add a date to automatically
+        unpublish a website jobs""",
+    'version': '10.0.1.0.0',
+    'license': 'AGPL-3',
+    'author': 'ACSONE SA/NV,Odoo Community Association (OCA)',
+    'website': 'https://github.com/OCA/website',
+    'depends': [
+        'hr',
```

Comment:
Remove dependence because if install `website_hr_recruitment` than autometic install `hr`

Reply:
@nikul-serpentcs IMO, we should not rely on transitive dependencies and keep this one.

Reply:
I don't think it's really important to mention the hr dependency as it included in the name of website_hr_recruitment, but I think for maintainability it's nicer to 'over explain'. So I have to agree with @ThomasBinsfeld on this one.

Reply:
@ThomasBinsfeld @tarteo here if install `website_hr_recruitment` then dependencies `hr_recruitment` and if it's install then automatic install `hr`, So no need to put `hr` in depends.
It's again check and nothing mean to add `hr`. 

Ex. https://www.odoo.com/documentation/10.0/reference/module.html